### DISCUSSION TITLE
Better todo list with indexes

### DISCUSSION THREAD

**Discussion by TomaszSzyborski** [zauberzeug/nicegui#534] [Q&A] [answered]
I tried to use the todo/trello cards from examples.
But the drag and drop feature effectively appends new cards as children in column.
Id like to have insert at specific index, like insert a card between cards.
I've found an example how to do it in quasar and vue here https://codesandbox.io/s/94yv5v6vrw?file=/index.js:405-447
is this behaviour repoducible in NiceGUI?

**Comment by falkoschindler** [comment, marked as answer]
In principle, this should be possible.
This discussion about inserting elements at a specific index might be relevant: https://github.com/zauberzeug/nicegui/discussions/489.
